From cd1c5e526da6154958553eea7bf439536a1f91e7 Mon Sep 17 00:00:00 2001
From: Zhizhong Zhang <zhizhong.zhang@amlogic.com>
Date: Thu, 25 Feb 2021 22:23:47 +0800
Subject: [PATCH] aml-mp-sdk: use staging dir instead of the relative dir [1/1]

PD#SWPL-test

Problem:
aml_mpd_sdk compile fail on openlinux

Solution:
use staging dir instead of the relative dir

Verify:
patch-build ok locally

Change-Id: I20e2363376ae7a494b8d2786a6c85b8abccf3d0d
---
 Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index b9cc438..1ab7a03 100644
--- a/Makefile
+++ b/Makefile
@@ -64,9 +64,7 @@ AML_MP_INC := $(AML_MP_ROOT_PATH)/include \
     $(AML_MP_ROOT_PATH)/ \
     $(AML_MP_ROOT_PATH)/tests/amlMpTestSupporter/ \
     $(AML_MP_ROOT_PATH)/tests/amlMpTestSupporter/demux/ \
-    $(AML_MP_ROOT_PATH)/../libdvr_release/include \
-    $(AML_MP_ROOT_PATH)/../mediahal_sdk/include \
-    $(AML_MP_ROOT_PATH)/../libplayer/src/amcodec/include \
+    $(STAGING_DIR)/usr/include/libdvr \
 
 CFLAGS += $(patsubst %,-I%,$(AML_MP_INC))
 LDFLAGS += -lmediahal_tsplayer -L. -lamdvr.product -llog -lcutils -lpthread -std=c++17 -D__unused=
-- 
2.29.2

